docs/user-interface/ui_overview.md

```diff
@@ -0,0 +1,106 @@
+---
+title: InstructLab UI Overview
+description: The overview of 🐶 InstructLab UI.
+logo: images/ilab_dog.png
+---
+
+The InstructLab User Interface (UI) is an additional component in the InstructLab project to make Knowledge/Skills 
+contributions more accessible to the general user. Instead of having to edit a `qna.yaml` file,
+users can fill out the contribution forms on the web. The user can also view their all of their contributions in a
+centralised list. You can also chat with the models directly on the playground.
+
+There are 2 ways to access the UI:
+
+1. Visit the website at [ui.instructlab.ai](https://ui.instructlab.ai/).
+
+2. Install and run the UI locally.
+
+
+!!! note
+    To log into the UI and submit Knowledge and Skills contributions, you must be a member of the [instructlab-public](https://github.com/instructlab-public) github repository.
+
+    If you aren't a member of the repository, but still wish to experiment with the UI, we recommend you to download it locally.
```

Comment:
I think we should recommend users to reach out to UI maintainers to add them to the instructlab-public org. If they don't want to join but still want to experiment, they can set it up locally.